server/appsettings.json

```diff
@@ -3,8 +3,8 @@
   "connections": {
     "database": {
       "name": "anon_grading_db",
-      "username": "sequelize",
-      "password": "_?aaD045.h",
+      "username": "root",
+      "password": "",
```

Comment:
revert this please. This is a security issue. We should never connect as a DBA through an ORM